<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceVueClass</class>
 <widget class="QMainWindow" name="FaceVueClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceVue</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="widgetTAB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="registrationTAB">
     <attribute name="title">
      <string>Registration</string>
     </attribute>
     <widget class="QLabel" name="registrationDisplayLBL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="recognitionTAB">
     <attribute name="title">
      <string>Recognition</string>
     </attribute>
     <widget class="QLabel" name="recognitionDisplayLBL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="loggingNameLBL">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>625</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>255</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>255</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="overlayCKB">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>630</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dispalay Overlays</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="FaceD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>625</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string> D</string>
      </property>
     </widget>
     <widget class="QLabel" name="FaceR">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>625</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> R</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>30</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="facevue.qrc">:/FaceVue/Resources/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QListWidget" name="imageslistLST">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>155</y>
      <width>261</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>List of Images</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagePreviewLBL">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>520</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBTN">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>670</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="facevue.qrc"/>
 </resources>
 <connections/>
</ui>
